resolwe/permissions/tests/test_data.py

```diff
@@ -69,6 +70,7 @@ def test_get_list_admin(self):
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         self.assertEqual(len(resp.data), 2)
 
+    @unittest.skipIf(True, 'since PR308: this test uses transactions, incompatible with the separated manager')
```

Comment:
This test should also work when signals in tests use a sync barrier?

Reply:
This was discussed some time ago. Getting rid of transactions here leads to all sorts of errors  in completely unrelated test support code which I chose to not deal with within the scope of this pull request, with consent from others.